src/views/TipCommentsView.vue

```diff
@@ -48,11 +48,13 @@
 
 <script>
 import { mapGetters } from 'vuex';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Can we avoid these somehow @mradkov ?

Reply:
I would ask about that too..

Reply:
They are not present in https://github.com/aeternity/aepp-raendom/pull/255/commits/81204d1b339216cbf0832f61fe3f0a009913fcf7 as I can see?

Reply:
-> 8f26461